.ci/Jenkinsfile

```
@@ -86,6 +86,24 @@ pipeline {
         }
       }
     }
+    /**
+     Execute integration tests.
+     */
+    stage('TestIntegration') {
+      steps {
+        deleteDir()
+        unstash 'source'
+        dir("${BASE_DIR}"){
+          sh(label: 'Runs the (integration) tests',script: '''
+            eval "$(gvm 1.14.2)"
+            go get -u github.com/magefile/mage
+            go get -u github.com/elastic/go-licenser
+            go get -u -v golang.org/x/tools/cmd/goimports
+            mage -debug testIntegration
+          ''')
+        }
+      }
```

This grabs the JUnit results

```suggestion
      }
      post {
        junit(allowEmptyResults: true, keepLongStdio: true, testResults: "junit-report.xml")
      }
```

---

wait, TestIntegration does not generate JUnit results like Test do

---

I wonder if it has to. I packed it into a golang test file but it could also be a binary that is executed through `go run .`.